code/books4u/utils.py

```diff
@@ -1,9 +1,19 @@
 from .models import *
+import re
 from django.contrib.sessions.backends.db import SessionStore
 from django.contrib.sessions.models import Session
 from django.utils import timezone
 
 
+def password_filter(pwd):
```

Comment:
unit test this.
It seems like it was not used in api.py?

Reply:
@not implemented yet. I will talk to front end team about return info